### ISSUE TITLE
Hook `Template::SubmissionWizard::Section::Review ` not inside `submissionWizard__reviewPanel`

### ISSUE THREAD

**Issue by henriquejsfj** [pkp/pkp-lib#9500]
**Describe the bug**
The Hook `Template::SubmissionWizard::Section::Review` called on the `'submission/wizard.tpl'` is supposed (I imagined) to permit devs to add a review to custom fields added in the steps. Although it is possible to add it, the visualization is not what is expected.

![image](https://github.com/pkp/pkp-lib/assets/16216517/aa34387c-7069-4b72-8cc9-2bb52e312185)

In the image above I added the check on `'editors'` step, however the fronted rendered it outside. This issue comes from the local where the hook is called. In the `'submission/wizard.tpl'` the reviewTemplate is rendered before closing the divs. 
```tpl
{foreach from=$reviewSteps item=$step}
    {if $step.reviewTemplate}
        {include file=$step.reviewTemplate}
    {/if}
    {call_hook name="Template::SubmissionWizard::Section::Review" submission=$submission step=$step.id}
{/foreach}
```

**To Reproduce**
I added this function to the Hook:
```php
function handleSubmissionReview($hookName, $args) {
    $step = $args[0]['step'];
    $templateMgr =& $args[1];
    $output = &$args[2];
    
    switch ($step) {
	    case 'editors':
		    $output .= $templateMgr->fetch($this->getTemplateResource('reviewEditors.tpl'));
		    break;
    }
    
    return false;
}
```
**What application are you using?**
 OJS version 3.4

**Additional information**
The `$output = &$args[2]` is coming with `null`value. I guess adding the result from reviewTemplate would be enough. Besides, it would be awesome if we have the new fields added to the form component generating it automatically.

**Comment by henriquejsfj** [comment]
Sorry, no PR this time, I don´t know exactly how to solve it.

**Comment by asmecher** [comment]
@henriquejsfj, if I understand right, basically you'd like to make a change to the markup but the hook you identified as the best candidate (`Template::SubmissionWizard::Section::Review`) is not at the right place in the markup for your needs. Do I have that right?

**Comment by henriquejsfj** [comment]
@asmecher Yes! In the print I sent I would like to have my field inside the Editor compartment.

I understand that the hook is useful when someone make a new step.

Besides, I can't use the TemplateManager's hooks since they are included in the tpl...

**Comment by jardakotesovec** [comment]
@henriquejsfj Hi, just to get bit more context. To display it in "For the editors" section - did you just added new field to the CommentsForTheEditors.php?

**Comment by henriquejsfj** [comment]
@jardakotesovec Yes!

I added a date like:
```php
$form->addField(new FieldText(BonoDate::DATE_TO_PUBLISH, [
    'label' => __('plugins.generic.bonoDate.formLabel'),
    'groupId' => 'default',
    'inputType' => 'date',
    'size' => 'large',
    'isRequired' => True,
    'value' => $form->publication->getData(BonoDate::DATE_TO_PUBLISH),
]));
```

**Comment by jardakotesovec** [comment]
@henriquejsfj Could you please try these changes - https://github.com/pkp/pkp-lib/pull/9513 And use new hook `Template::SubmissionWizard::Section::Review::ForTheEditors` to verify you can achieve what you described?

**Comment by henriquejsfj** [comment]
@jardakotesovec Solved! Thanks so much!
![image](https://github.com/pkp/pkp-lib/assets/16216517/66247e03-baab-4993-b03e-e98303d98d4f)

**Comment by asmecher** [comment]
(@jardakotesovec, please feel free to sneak these into the upcoming release -- but if you do, make sure to cherry-pick as needed and update the milestone here to the lowest version number that will include the changes!)

**Comment by jardakotesovec** [comment]
@henriquejsfj @asmecher 
Oki, added hooks also for other review files, and adjusted naming to be more consistent with the files.

Please not that the hook name now is: `Template::SubmissionWizard::Section::Review::Editors`

Its now merged in 3.4 and main

**Comment by henriquejsfj** [comment]
Nice @jardakotesovec, thanks!
